src/dotnet-install.sh

```diff
@@ -834,7 +835,7 @@ install_dotnet() {
     #  if the download fails, download the legacy_download_link
     if [ "$download_failed" = true ]; then
         say "Cannot download: $download_link"
-
+		rm -f "$zip_path" 2>&1 && say_verbose "Temporary zip file $zip_path was removed or doesn't exist"
```

Comment:
"...was removed..." sounds like as if the operation succeeded. Also, if we put the path to the end, the logs would be easier to read if the path is long. Such as:
```
Failed to cleanup the temporary file at location: $zip_path
```

Reply:
this message actually indicates if removal is successful, but we don't know if file existed before or not. It will be displayed only if rm exists with 0 code.
error message (if any) will be redirected.

Reply:
Then the user probably does not care :) If the file is not there, it does not take any space and there is no need to confuse the user with this message.